diff --git a/chrome/browser/ui/views/frame/browser_view_layout.cc b/chrome/browser/ui/views/frame/browser_view_layout.cc
index 7359b0457bf17..f5b804366b429 100644
--- a/chrome/browser/ui/views/frame/browser_view_layout.cc
+++ b/chrome/browser/ui/views/frame/browser_view_layout.cc
@@ -73,6 +73,7 @@ bool ConvertedHitTest(views::View* src, views::View* dst, gfx::Point* point) {
   DCHECK(src);
   DCHECK(dst);
   DCHECK(point);
+  BRAVE_BROWSER_VIEW_LAYOUT_CONVERTED_HIT_TEST
   views::View::ConvertPointToTarget(src, dst, point);
   return dst->HitTestPoint(*point);
 }
@@ -262,7 +263,7 @@ gfx::Size BrowserViewLayout::GetMinimumSize(const views::View* host) const {
   }
 
   // The minimum height for the normal (tabbed) browser window's contents area.
-  constexpr int kMainBrowserContentsMinimumHeight = 1;
+  constexpr int kMainBrowserContentsMinimumHeight = 2;
 
   const bool has_tabstrip =
       delegate_->SupportsWindowFeature(Browser::FEATURE_TABSTRIP);
